lib/boot.js

```diff
@@ -1,3 +1,19 @@
+const foundElectron = window.process && window.process.type;
+
+if (!foundElectron) {
+  if (localStorage.getItem('appOpened')) {
+    const message = document.createElement('h1');
+    message.innerText =
+      'Simplenote cannot currently be opened simultaneously in more than one tab or window per browser';
+    document.body.appendChild(message);
+    throw new Error('Simplenote can only be opened in one tab');
+  }
+  localStorage.setItem('appOpened', true);
+  window.addEventListener('unload', function(event) {
+    localStorage.removeItem('appOpened');
+  });
+}
```

Comment:
it might be helpful to split this into a separate module to keep the logic separate and avoid the noise…

```js
import './ensure-single-browser-tab-only';
import 'core-js/stable';
import 'regenerator-runtime/runtime';
…
```

Reply:
Done.